deployment/updates/scripts/2022_01_05_admin_tags_equals_boolean_notification.php

```diff
@@ -0,0 +1,14 @@
+<?php
+/**
+ * @package deployment
+ */
+require_once (__DIR__ . '/../../bootstrap.php');
+$script = realpath(dirname(__FILE__) . "/../../../tests/standAloneClient/exec.php");
+$newTemplateUpdate = realpath(dirname(__FILE__) . "/../../updates/scripts/xml/notifications/2022_01_05_admin_tags_equals_boolean_notification.template.xml");
```

Comment:
need to remove .template here like in the rest pf the deployment scripts

Reply:
Good catch. Fixing.